client/src/components/Form/index.style.js

```diff
@@ -3,6 +3,9 @@ import { Form, Field } from "formik";
 import "react-datepicker/dist/react-datepicker.css";
 import DatePicker from "react-datepicker";
 
+export const StyledContainerBtn=styled.div`
+margin-right:422px;
```

Comment:
This isn't the way to place items of the page! Look into using flex or absolute positioning.